app.py

```diff
@@ -431,12 +450,11 @@ def attendence_post():
 
 @socketio.on('join', namespace='/rooms_namespace')
 def on_join(data):
-    debug(str(data))
-    debug(str(type(data)))
-    data = json.loads(data)
-    #verify if attendence is success
-    #then get them join in that room name with otp added
-    #now broadcast to that room
+    """verify if attendence is success 
+    then get them join in that room name with otp added
+    now broadcast to that room
+    """
+    data = json.loads(data) # now it's dict form, before it was string
```

Comment:
Handle exception here

Reply:
@L04DB4L4NC3R what exception can occur here? socket will be active only when data comes. and data will be string which is truned into dict.
I am adding a catch block though.

Reply:
During testing, It will break if @Nirmitjatana  is sending something other than a streing. Plus, Its better to send JSON rather than just string data

Reply:
It should be json only. I found it to be streing, will see again for reason (previously didn't bother much though am sending json only but in case it's coming streing for this).
It will be hardcoded in code so user cann't break it. Nirmit is not sending on this namespace (and even if did then it would be mistake to send integer for dict value)
on this namespace admin flag will decide and send data.

I have fixed it though as you said.
this socket creates rooms